.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- Good work on the project so far, but you need to implement one necessary change. PLease add the node module into the `.gitignore` as it is a requirement in the project.

Reply:
Thanks @Ginohmk. Do I need to create the node module directory or I can just node module into the .gitignore?

Reply:
Hi @inspecta Please just add the node module, to the `.gitignore`  so it becomes a second nature to add it to a ignore file, Also your project should have included a node module, never the less, I would suggest you just add it to git ignore and request for a review.